iris/iris.go

```diff
@@ -167,7 +167,17 @@ func GetResponse(res *http.Response, v interface{}) error {
 		return err
 	}
 
-	err = mapstructure.Decode(j, &v)
+	config := &mapstructure.DecoderConfig{
+		WeaklyTypedInput: true,
+		Result:           &v,
```

Comment:
this doesn't use the `json` tag

Reply:
Which `json` tag and where?

Reply:
you users would pass structs that use `json` tags to define how to encode/decode the struct into JSON. This library supports customisation to that effect.